test/lit/passes/dae-gc-refine-params.wast

```diff
@@ -35,37 +35,46 @@
   ;; all nulls are identical and we could do other optimization work due to
   ;; that.
   (call $various-params-no
-   (call $get_{})
-   (call $get_{i32})
+   (call $"get_{}")
+   (call $"get_{i32}")
   )
   (call $various-params-no
-   (call $get_{i32})
-   (call $get_{f64})
+   (call $"get_{i32}")
+   (call $"get_{f64}")
   )
  )
  ;; This function is called in ways that do not allow us to alter the types of
  ;; its parameters (see last function).
- ;; CHECK:      (func $various-params-no (type $7) (param $x (ref null ${})) (param $y (ref null ${}))
+ ;; CHECK:      (func $various-params-no (type $7) (param $x (ref null $"{}")) (param $y (ref null $"{}"))
  ;; CHECK-NEXT:  (drop
  ;; CHECK-NEXT:   (local.get $x)
  ;; CHECK-NEXT:  )
  ;; CHECK-NEXT:  (drop
  ;; CHECK-NEXT:   (local.get $y)
  ;; CHECK-NEXT:  )
  ;; CHECK-NEXT: )
- (func $various-params-no (param $x (ref null ${})) (param $y (ref null ${}))
+ (func $various-params-no (param $x (ref null $"{}")) (param $y (ref null $"{}"))
   ;; "Use" the locals to avoid other optimizations kicking in.
   (drop (local.get $x))
   (drop (local.get $y))
  )
 
- (func $get_{} (result (ref null ${}))
+ ;; CHECK:      (func $"get_{}" (type $8) (result (ref null $"{}"))
+ ;; CHECK-NEXT:  (unreachable)
+ ;; CHECK-NEXT: )
+ (func $"get_{}" (result (ref null $"{}"))
```

Comment:
Why was there no `CHECK` for this function before?

Reply:
This file doesn't use `--all-items`, so when the name doesn't match, the output doesn't appear at all.